support-tos.md

```diff
@@ -22,21 +22,40 @@ If specified at time of purchase, a feature key entitles the holder to priority
 
+## Billing Options
+
+We offer two billing plans:
+
+* **Prepaid annual billing:** You will be charged at the time of purchase for a feature key covering the number of users you specify and valid for 365 days. You may increase the user limit at any time. If you increase the user limit, you will be charged for the new users prorated for the time remaining in the billing cycle. If you decrease the user limit, the change will take effect upon your next renewal.
+
+* **Usage-based monthly billing:** Every 30 days, you will be charged according to the actual number of active users seen by your server in the last 30 days, not counting "visitor" users. Your server reports the number of users automatically. You may inspect your server's active user count by visiting the "statistics" section of the administrator settings. You must not manipulate your server in a way that causes it to report an incorrect count.
+
 ## Automatic Renewal
 
-If you purchase a feature key with automatic renewal, your Sandstorm server will automatically download a new key when your first key expires, and you will automatically be charged. To cancel renewal, you may do *any* of the following:
+At the end of each billing period, your Sandstorm server, if it is still running, will automatically renew your key, and you will automatically be billed. To cancel renewal, you may do either of the following:
 
 * Permanently shut down your Sandstorm server, in which case it will not renew the key.
-* De-configure the feature key in your server's settings.
-* Contact [priority-support@sandstorm.io](mailto:priority-support@sandstorm.io) and request that we cancel the renewal policy.
+* Delete the feature key from your server's configuration.
+
+If you cannot do one of the above because you have lost control of your key or your server, you may contact [priority-support@sandstorm.io](mailto:priority-support@sandstorm.io) and request that we revoke your key.
+
+## Free Trials
+
+Your organization may be eligible for a free trial, as indicated in the feature key purchase tool.
+
+During the trial period, you agree to share basic server statistics with Sandstorm, such as the number of active users and which apps they are using, to help our sales team determine the best options for you. To review the statistics being shared, visit the "statistics" page in the admin panel of your server (i.e. `<your-server>/admin/stats`). Once you've purchased your feature key, you may choose not to continue sharing this information.
+
+## Violations
+
+If we conclude, at our sole discretion, that you have violated these terms, we may revoke your feature key, meaning that your Sandstorm server may cease to recognize it and Sandstorm for Work features may stop functioning. We will notify you by e-mail if we take this action.
```

Comment:
I assume they don't get a refund if they have a ToS violation? Does that need to be spelled out?